unittest/python/CMakeLists.txt

```diff
@@ -108,6 +108,11 @@ if(Python_EXECUTABLE)
            WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
   set_tests_properties(PythonFormats PROPERTIES ENVIRONMENT "${PYTHON_TEST_ENVIRONMENT}")
 
+  add_test(NAME PythonPizza
+           COMMAND ${PYTHON_TEST_RUNNER} ${CMAKE_CURRENT_SOURCE_DIR}/python-pizza.py -v
+           WORKING_DIRECTORY ${EXECUTABLE_OUTPUT_PATH})
+  set_tests_properties(PythonPizza PROPERTIES ENVIRONMENT "${PYTHON_TEST_ENVIRONMENT}")
```

Comment:
How about extending `PYTHONPATH` in `PYTHON_TEST_ENVIRONMENT` to include the pizza directory? Instead of adding the relative path in `python-pizza.py`?

Reply:
Could be done. But on the other hand, this is closer to the suggested way of using the Pizza.py files.